plugin/pkg/auth/authorizer/rbac/bootstrappolicy/all.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package bootstrappolicy
+
+import "k8s.io/kubernetes/pkg/apis/rbac"
+
+type PolicyData struct {
```

Comment:
I prefer this struct in the registry package with the `PostStartHook` method (possibly called `EnsureRBACPolicy`) as a method using this as a receiver.